packages/components/src/index.ts

```diff
@@ -132,6 +132,7 @@ export { default as PanelHeader } from './panel/header';
 export { default as PanelRow } from './panel/row';
 export { default as Placeholder } from './placeholder';
 export { default as Popover } from './popover';
+export { ProgressBar } from './progress-bar';
```

Comment:
Maybe we could use `default as ProgressBar` to be consistent with the rest.

Reply:
Sure, it'd be easier on the eyes.

Reply:
Done in https://github.com/WordPress/gutenberg/pull/61062/commits/2f978f4af873993a7aa2bea0c8494bb368d2453a.